app/com/linkedin/drelephant/spark/heuristics/StagesHeuristic.scala

```diff
@@ -116,6 +129,15 @@ object StagesHeuristic {
     critical = Duration("60min").toMillis,
     ascending = true
   )
+  
+  /** The severity thresholds for the ratio of JVM GC Time and executor CPU Time */
```

Comment:
Can you add a comment on what basis these thresholds were set? If it is experimental, please mention it in the comments and that it may change in the future.

Reply:
These are experimental, so are likely to change as we evaluate the results on more Spark applications.